Added init( ) to code.

vishaaaal-droid opened the pull request:
Kindly accept this PR,  adding init to the code.

vishaaaal-droid committed:
Added init function.

vishaaaal-droid committed:
Create main.cpp

vishaaaal-droid committed:
Add files via upload

vishaaaal-droid committed:
Update Readme.md

vishaaaal-droid committed:
Create build.sh

vishaaaal-droid committed:
Add files via upload

vishaaaal-droid committed:
Create addLog

vishaaaal-droid committed:
Add files via upload

vishaaaal-droid committed:
Create commitLog

vishaaaal-droid committed:
Delete addLog

vishaaaal-droid committed:
Delete commitLog

vishaaaal-droid committed:
Delete conflictlog

vishaaaal-droid committed:
Delete .imperiumignore

vishaaaal-droid committed:
Create addLog

vishaaaal-droid committed:
Create commitLog

vishaaaal-droid committed:
Add files via upload

vishaaaal-droid committed:
Add files via upload

vishaaaal-droid commented:
Added by Vishal Mahavar.

v-rohan commented:
Kindly remove the .imperium folder and .imperiumIgnore files from your commits. Add them to the gitIgnore if you're using CLI, and make another commit. This should only have your project files.

vishaaaal-droid commented:
Okay, understood.

vishaaaal-droid committed:
Delete addLog

vishaaaal-droid committed:
Delete commitLog

vishaaaal-droid committed:
Delete conflictlog

vishaaaal-droid committed:
Delete .imperiumignore

v-rohan commented:
Please make sure the files are in a folder with your username.

vishaaaal-droid commented:
Yes, I will do.

vishaaaal-droid committed:
Create README.md

vishaaaal-droid committed:
Rename src/main.cpp to vishaaaal-droid/src/main.cpp

vishaaaal-droid committed:
Rename script/build.sh to vishaaaal-droid/script/build.sh

vishaaaal-droid committed:
Rename script/imperium.sh to vishaaaal-droid/script/imperium.sh

vishaaaal-droid committed:
Rename Makefile to vishaaaal-droid/Makefile

vishaaaal-droid committed:
Rename goorm.manifest to vishaaaal-droid/goorm.manifest

vishaaaal-droid committed:
Delete README.md

vishaaaal-droid committed:
Update Readme.md

vishaaaal-droid commented:
I hope it's fine now. Please tell me if I need to make some other changes, I'll do that.

v-rohan commented:
No, it's fine.

vishaaaal-droid committed:
Fixed some typo in main.cpp

Changed commitlog to commit.log, conflictlog to conflict.log and addlog to add,log,

vishaaaal-droid committed:
Fixed typo in main.cpp

vishaaaal-droid committed:
Update Readme.md
